configs.js

```diff
@@ -0,0 +1,42 @@
+let alert = document.getElementById('alert');
+
+function showAlert(){
+       alert.style.display = 'block'
+    setTimeout(function () {
+        alert.style.display = 'none'
+    }, 20000)
+}
+
+function restoreOptions() {
+    chrome.storage.sync.get(
+        'mode',
+        function (items) {
```

Comment:
I prefer if anonymous/callback functions always have a name or to be separated it into it's new function for easier debugging and readability 

```chrome.storage.sync.get('mode', function onStorageGet() {})``` etc ...